kod/object/item/passitem/numbitem/food/gobletwn.kod

```diff
@@ -44,6 +44,7 @@ properties:
    viFilling = 8
    viNutrition = 6
    piNumber = 1
+   piItem_flags = 50    % Pewter color to set it apart from goblet of ale
```

Comment:
Please use the symbolic names from blakston.khd rather than hard-coding the values.

Reply:
Alright, although some of them don't have entries, such as the silvery color applied to gobletwn.kod.  Should I add them to blakston.khd?